lib/client-assets.php

```diff
@@ -574,3 +574,50 @@ function gutenberg_register_vendor_scripts( $scripts ) {
 // Enqueue stored styles.
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_stored_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_stored_styles', 1 );
+
+/**
+ * Registers interactivity scripts for Gutenberg.
+ *
+ * This function registers interactivity scripts for Gutenberg when not in the
+ * admin panel.
+ */
+function gutenberg_register_interactivity_scripts() {
+ 	if ( ! is_admin() )	 {
+		wp_register_script(
+			'interactivity-runtime',
+			plugins_url(
+				'../build/block-library/interactive-blocks/interactivity.min.js',
+				__FILE__
+			),
+			array( 'interactivity-vendors')
+		);
+
+		wp_register_script(
+			'interactivity-vendors',
+			plugins_url(
+				'../build/block-library/interactive-blocks/vendors.min.js',
+				__FILE__
+			)
+		);
+ 	}
+}
+add_action( 'wp_enqueue_scripts', 'gutenberg_register_interactivity_scripts' );
+
+/**
+  * Adds the "defer" attribute to all the interactivity script tags.
+  *
+  * @param string $tag The generated script tag.
+  * @param string $handle The script's registered handle.
+	*
+  * @return string The modified script tag.
+	*/
+function add_defer_attribute( $tag, $handle ) {
+		if ( 0 === strpos( $handle, 'interactivity-' ) ) {
```

Comment:
Correct, they will be manually registered with the same prefix, at least for now. Maybe it would be a good idea to make the prefix less "generic", like `wp-interactivity-api-`, or similar.

Reply:
Updated to `wp-interactivity-` prefix for now as it is unique enough. See https://github.com/WordPress/gutenberg/pull/49994/commits/6ae760f5e92147285da1cf8dbb8aa44ce8264272.